src/EventListener/MergeHttpHeadersListener.php

```diff
@@ -92,8 +92,10 @@ private function mergeHttpHeaders(Response $response)
                 header_remove($name);
             }
 
-            // Do not replace existing headers as the response object has a higher priority
-            $response->headers->set($name, trim($content), false);
```

Comment:
Not replacing it means it adds it. You end up having e.g. two `Content-Type` header which is illegal and will not work properly.